app/javascript/stylesheets/_sidebar.scss

```diff
@@ -11,3 +11,11 @@
 #sidebar-wrapper .list-group {
   width: 15rem;
 }
+
+#sidebar-wrapper .sidebar-footer {
+  padding: 0.875rem 1.25rem;
+  font-size: 0.8rem;
+  position: fixed;
```

Comment:
I think bootstrap has pseudo classes for these things e.g. `position: fixed`
Would it also be possible to use one of the predefined padding values from Bootstrap? https://getbootstrap.com/docs/4.0/utilities/spacing/

Same with font-size.

Reply:
I added pseudo classes for `padding` and `position-fixed`, but haven't found the same for font-size.. As I can see, in Zollinger project you explicitly defined font-size in `.scss` files

Reply:
I'm just curious, do you need to change the font size for the books table only? Or would it be okay to customise the font size of the `<p></p>` tags in general (and therefore have it consistent throughout the app)?